addons/necropost-highlighter/addon.json

```diff
@@ -0,0 +1,142 @@
+{
+  "name": "Necropost highlighter",
+  "description": "Spot old forum topics brought back to life, before opening the message. Topics started much earlier than others listed on the same page are highlighted (in selected forums). If you've wasted time reading zombie messages or a help/question/request that was actually asked years ago, this is for you!",
+  "info": [
+    {
+      "type": "notice",
+      "id": "whatGetsHighlighted",
+      "text": "Sometimes an older topic is kept alive for good reasons, like an ongoing conversation, or the original poster following up their own topic months later. All older topics in selected forums are highlighted, including occasional lingering topics that are not necroposts. (That's why they get highlighted, but not removed.)"
+    }
+  ],
+  "credits": [
+    {
+      "name": "RT_Borg",
+      "link": "https://github.com/RT-Borg"
+    }
+  ],
+  "dynamicDisable": true,
+  "dynamicEnable": true,
+  "enabledByDefault": false,
+  "tags": ["community", "forums"],
+  "versionAdded": "1.28.0",
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["^https://scratch.mit.edu/discuss/(?:m/|)(?:search|1|3|4|6|7|9|11|49)/"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "style.css",
+      "matches": ["^https://scratch.mit.edu/discuss/(?:m/|)(?:search|1|3|4|6|7|9|11|49)/"]
+    }
+  ],
+  "settings": [
+    {
+      "name": "Roughly how many months is 'old'?",
+      "id": "monthCountConsideredOld",
+      "type": "positive_integer",
+      "default": 1
+    },
+    {
+      "name": "Color topic cells?",
+      "id": "colorTopicCells",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "name": "Choose custom forums?",
+      "id": "chooseCustomForums",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "name": "New Scratchers (usually&nbsp;accurate)",
```

Comment:
Is `&nbsp;` necessary? It makes the code a little harder to read.

Reply:
It's been quite a few months, but as I recall, I added the non-breaking space after recognizing that the output looked really bad when "(usually" formats on one line and "accurate)" overflows to the next line. I suggest keeping the non-breaking space unless the review team wants to test it the other way and disagrees about how qualifiers like "(usually accurate)" look when split.